src/main/java/xyz/duncanruns/julti/JultiOptions.java

```diff
@@ -147,6 +147,9 @@ public final class JultiOptions {
     public boolean resizeableBorderless = false;
     // public boolean forceActivate = false;
 
+    // Launching
+    public List<Path> launchingProgramPaths = new ArrayList<Path>();
```

Comment:
Interesting, I didn't know `Path` objects worked with GSON, I would have used this with the instance paths saving